plugins/hiddenore-paper/src/main/java/com/github/devotedmc/hiddenore/util/FakePlayer.java

```diff
@@ -2253,6 +2253,11 @@ public Spigot spigot() {
         return null;
     }
 
+    @Override
```

Comment:
hiddenore stuff in bastion pr?

Reply:
yeah it was necessary to get this to build. But I deleted this class anyway so it doesn't matter